crunch-hbase/src/main/java/org/apache/crunch/io/hbase/HFileUtils.java

```diff
@@ -384,6 +451,24 @@ public void process(Pair<ByteBuffer, Iterable<C>> input, Emitter<Result> emitter
     writeToHFilesForIncrementalLoad(cells, connection, tableName, outputPath, false);
   }
 
+  public static <C extends Cell> void writeToHFilesForIncrementalLoad(
```

Comment:
javadoc for these 2 new public methods?

Reply:
added, https://github.com/noslowerdna/crunch/commit/9dd88da343727acd4685e220f499f4fc21770dfe